tests/test_power.py

```diff
@@ -0,0 +1,72 @@
+"""Tests for the _ATEN_PE6108G PDU class"""
+import time
+
+import mock
+from nose.tools import eq_, raises
+from pysnmp.entity.rfc3413.oneliner import cmdgen
+from pysnmp.proto.rfc1902 import Integer
+
+from _stbt.power import uri_to_power_outlet
+
+
+def mock_data(int_value):
+    """Match the format of the data returned from pysnmp"""
+    return (None, None, None, [(oid, Integer(int_value))])
+
+
+cmdgen.UdpTransportTarget = mock.Mock()
+time.sleep = mock.Mock()
+
+outlet = 1
+oid = "1.3.6.1.4.1.21317.1.3.2.2.2.2.{0}.0".format(outlet + 1)
+aten = uri_to_power_outlet('aten:mock.host.name:1')
```

Comment:
Please implement setup as a `@contextmanager`.  With this as it is the code will be run at import time rather than before each test and the same aten object could be shared between each of the tests.  This should allow you to tidy-up the mock patching too.

Something like:

    @contextmanager
    def mocked_aten():
        cmdgen.UdpTransportTarget = mock.Mock()
        time.sleep = mock.Mock()

        outlet = 1
        oid = "1.3.6.1.4.1.21317.1.3.2.2.2.2.{0}.0".format(outlet + 1)
        aten = uri_to_power_outlet('aten:mock.host.name:1')

        with mock.patch('pysnmp.entity.rfc3413.oneliner.cmdgen.CommandGenerator') as mock_class:
            yield (mock_class.return_value, aten)

Although I'd still be a little concerned about the side effects of `time.sleep = mock.Mock()`.  Should that also be `mock.patch('time.sleep')`, so `time.sleep` is not permanently mocked'?